app/views/StartPageView.scala.html

```diff
@@ -0,0 +1,79 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+
+@this(layout: Layout)
+
+@(isAgent: Boolean)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@mainHTML = {
+<main class="govuk-main-wrapper">
+ <div class="govuk-width-container">
+
+  <span class="govuk-caption-xl">@messages("startPage.taxYear")</span>
+
+ @if(isAgent){
+  <h1 id = "agentHeader" class="govuk-heading-xl">@messages("startPage.heading.agent")</h1>
+
+
+  <div class="govuk-body">
+   <p>@messages("startPage.agent1")</p>
+   <p>@messages("startPage.agent2")</p>
+  </div>
+
+  <ul class="govuk-list govuk-list--bullet">
+   <li>@messages("startPage.interest")</li>
+   <li>@messages("startPage.dividends")</li>
+  </ul>
+
+  <div class="govuk-body">
+   <p>@messages("startPage1")</p>
+   <p>@messages("startPage2")</p>
+  </div>
+
+  <button class="govuk-button" data-module="govuk-button">
+   @messages("startPage.provideUpdates")
+  </button>
+
+ } else {
+  <h1 class="govuk-heading-xl">@messages("startPage.heading.individual")</h1>
+
+  <div class="govuk-body">
+   <p>@messages("startPage.individual1")</p>
+   <p>@messages("startPage.individual2")</p>
+ </div>
+
+  <ul class="govuk-list govuk-list--bullet">
+  <li>@messages("startPage.interest")</li>
+  <li>@messages("startPage.dividends")</li>
+ </ul>
+
+  <div class="govuk-body">
+   <p>@messages("startPage1")</p>
+   <p>@messages("startPage2")</p>
+  </div>
+
+  <button class="govuk-button" data-module="govuk-button">
+   @messages("startPage.provideUpdates")
+  </button>
+ }
+  </div>
+</main>
+}
+
+ @layout(pageTitle = Some("income-tax-submission-frontend"))(mainHTML)
```

Comment:
no indentation required here, we should also be using messages for the string val here(service.name). Please see the messages file comment for content recommendation. We will need to also have a discussion about separating the title and the govuk header banner heading as these should be separate, the title should be in the following format = [{Page heading/name} - {Service Name} - GOV.UK]